tests/unit/options.js

```diff
@@ -1413,7 +1416,16 @@ exports.esnext = function (test) {
     'const foo = 9;',
     'var myConst = function (test) { };',
     'foo = "hello world";',
-    'var a = { get x() {} };'
+    'var a = { get x() {} };',
+    'if (a) {',
+    '  const bar = 3;',
+    '  bar++;',
+    '  bar+=1;',
```

Comment:
Add some tests with different binary operations, eg

```js
const f = () => 1;
bar();
const a = [1, 2, 3];
a[0] = 2;
const o = { a: 1 };
o.a = 2;
```

things like that (all of these are legal)